vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.h

```diff
@@ -3,17 +3,50 @@
 
 #pragma once
 
+#include <INativeUIManager.h>
+#include <ShadowNodeBase.h>
+#include <UI.Xaml.Documents.h>
+#include <Utils/TransformableText.h>
 #include <Views/FrameworkElementViewManager.h>
 
 namespace react::uwp {
 
+struct VirtualTextShadowNode final : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+  TransformableText m_transformableText{};
+  void AddView(ShadowNode &child, int64_t index) override {
+    auto &childSNB = static_cast<ShadowNodeBase &>(child);
```

Comment:
Nit: could you add whitespace before the start of the function? Some of this is a bit hard to parse out as this large of a block.